### DISCUSSION TITLE
Restoring existing network

### DISCUSSION THREAD

**Discussion by boesing** [Koenkk/zigbee2mqtt#5592] [General]
Hey there,

I've temporarily unplugged my ZZH Stick and re-plugged CC2531 as I wanted to reset a philips hue bulb with touchlink which is not supported for the ZZH stick.

I've changed PAN ID and some other ZZH related settings back to default and could sucessfully reset the Hue Bulb with my CC2531.
After this, I've restored configuration from a snapshot, re-plugged the ZZH stick and restarted my host.

After starting up zigbee2mqtt again, all devices had "No Route to host" so I somehow totally screwed up my Zigbee Network (around 20 devices atm).

Is there any "trick" to restore the existing network without having to repair **ALL** devices?

Thanks in advance.

**Comment by Koenkk** [comment]
> After starting up zigbee2mqtt again, all devices had "No Route to host"

Usually can be fixed by re-powering the routers (shortly disconnect them from mains)

**Reply by boesing** [reply]
Sadly that didn't work but I restored a snapshot which restores access to most of my devices.
